pkg/amazon/backend/convert.go

```diff
@@ -173,6 +182,48 @@ func Convert(project *types.Project, service types.ServiceConfig) (*ecs.TaskDefi
 	}, nil
 }
 
+func createEnvironment(project *types.Project, service types.ServiceConfig) ([]ecs.TaskDefinition_KeyValuePair, error) {
+	environment := map[string]*string{}
+	for _, f := range service.EnvFile {
+		if !filepath.IsAbs(f) {
+			f = filepath.Join(project.WorkingDir, f)
+		}
+		if _, err := os.Stat(f); os.IsNotExist(err) {
```

Comment:
Any reason we don't return if there is any stat error? Do we expect the `Open` to fail for all others?

It would be good to check that it's not a folder too. (i.e. `s, err := os.Stat(f); s.IsDir() || err != nil`)

Reply:
This is the same code using in compose-go. I'm indeed expecting `os.Open` to fail for other errors.